RootTools/python/samples/triggers_13TeV_DATA2016.py

```diff
@@ -8,6 +8,8 @@
 triggers_mumu = triggers_mumu_iso
 
 triggers_ee = [ "HLT_Ele17_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v*", "HLT_Ele23_Ele12_CaloIdL_TrackIdL_IsoVL_DZ_v*" ] # 17/12 prescaled in column 0
+triggers_ee_noniso = ["HLT_DoubleEle33_CaloIdL_v*", "LT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v*", "HLT_DoubleEle33_CaloIdL_GsfTrkIdVL_v" ]
```

Comment:
I think "LT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v*" should be  "HLT_DoubleEle37_Ele27_CaloIdL_GsfTrkIdVL_v*"